shuup/admin/modules/orders/views/detail.py

```diff
@@ -82,6 +84,9 @@ def get_toolbar(self):
         ))
 
         for button in get_provide_objects("admin_order_toolbar_button"):
+            warnings.warn(
+                "admin_order_toolbar_button provider is deprecated, use admin_order_toolbar_action_item instead",
+                DeprecationWarning, stacklevel=2)
```

Comment:
will setting stacklevel=2 actually have the intended effect here? isn't the caller in this case something internal to django?